docs_user/modules/openstack-tls_everywhere.adoc

```diff
@@ -0,0 +1,211 @@
+[id="tls-everywhere_{context}"]
+
+//:context: tls
+
+//kgilliga: This module will be converted to an assembly. Check xref contexts.
+
+= TLS Everywhere
+
+_Disclaimer: the below steps were reproduced on a FreeIPA 4.10.1 server. The location of files and directories may slightly change on different versions._
+
+== Prerequisites
+
+* Check that the source deployment is using TLS Everywhere, skip this step if it's not.
+* Make sure the previous Adoption steps have been performed successfully.
+* Make sure the backend services on the new deployment are not started yet.
+
+== Variables
+
+Define the shell variables used in the following steps. The values are illustrative and refer to a single node standalone director deployment. Use values that are correct for your environment:
+
```

Comment:
This needs to be conditional or it will fail the build.

----
ifeval::["{build}" != "downstream"]
IPA_SSH="ssh -i ~/install_yamls/out/edpm/ansibleee-ssh-key-id_rsa root@192.168.122.100 podman exec -ti freeipa-server-container"
endif::[]
ifeval::["{build}" == "downstream"]
IPA_SSH="ssh -i <path_to_ssh_key> root@<freeipa-server-ip-address>"
endif::[]

Reply:
This looks weird in the formatting.  Ping me for details.